tutorials/sinon/mock.md

```diff
@@ -0,0 +1,13 @@
+Mocks allow you to create a fake function that passes or fails depending on your needs.
+You can ensure it was called with certain arguments, or check how many times it was called.
```

Comment:
Add another sentence about how you need to call mock() on an object, and that you need to define expectations up front (https://semaphoreci.com/community/tutorials/best-practices-for-spies-stubs-and-mocks-in-sinon-js)